libraries/ESP8266HTTPClient/examples/DigestAuthorization/DigestAuthorization.ino

```diff
@@ -0,0 +1,117 @@
+#include <ESP8266WiFi.h>
+#include <ESP8266WiFiMulti.h>
+
+#include <ESP8266HTTPClient.h>
+
+#define USE_SERIAL Serial
+
+ESP8266WiFiMulti WiFiMulti;
+
+String _exractParam(String& authReq, const String& param, const char delimit){
+	int _begin = authReq.indexOf(param);
```

Comment:
Please load this sketch into Arduino IDE and use Tools > Auto Format feature to fix the indentation.

Reply:
Also, it is not a common practice to prefix function and field names with an underscore, when not implementing class members, at least in Arduino.